Nodejs/Product/TestAdapter/TestFrameworks/Angular/Angular.js

```diff
@@ -17,16 +17,17 @@ function getTestProjects(configFile) {
     for (const projectName of Object.keys(angularConfig.projects)) {
         const project = angularConfig.projects[projectName];
 
-        const karmaConfigPath = project.architect.test
+        const karmaConfigFilePath = project.architect.test
             && project.architect.test.options
             && project.architect.test.options.karmaConfig
             && path.resolve(configPath, project.architect.test.options.karmaConfig);
 
-        if (karmaConfigPath) {
+        if (karmaConfigFilePath) {
             angularProjects.push({
                 angularConfigPath: configPath,
-                karmaConfigPath,
+                karmaConfigFilePath,
                 name: projectName,
+                rootPath: path.join(configPath, project.root),
```

Comment:
Why `path.join`, rather than `path.resolve`, as above?

Reply:
No particular reason, I think for this scenario both will work ok.

My first though is that we just need to merge the paths since `configPath` will be a full path and `root` will be relative; so I just went with join.